Rakefile

```diff
@@ -4,3 +4,18 @@
 require File.expand_path('../config/application', __FILE__)
 
 Rails.application.load_tasks
+
+# Create a new DB backup, and import it to the local Postgres DB
+task :db do
+  app = ENV["HEROKU_APP"] || "fastlane-refresher"
+  db_name = ENV["DB_NAME"] || "refresher"
+
+  puts "This script is going to drop your local database #{db_name} and fetch the database from heroku #{app}. Please confirm!"
+  STDIN.gets
```

Comment:
Yes, that would be nicer, however I didn't initially want to spend the time looking for how to do that, as we don't have highline or similar as dependency, related https://twitter.com/KrauseFx/status/775499416878788609 👍